ci/mac_ci_setup.sh

```diff
@@ -9,6 +9,15 @@ set -e
 # https://github.com/actions/virtual-environments/blob/master/images/macos/macos-10.15-Readme.md for
 # a list of pre-installed tools in the macOS image.
 
+export ANDROID_HOME=$HOME/Library/Android/sdk
```

Comment:
Do you mind moving to the bottom of the file and leaving a comment stating why this is done?

Also do you mind cleaning CI workflows that set any of these variables. Now that they are exported here they don't need to be set elsewhere.

Reply:
>Do you mind moving to the bottom of the file and leaving a comment stating why this is done?
yup, it's planned, it's at the top for debugging - to get executed first

>Also do you mind cleaning CI workflows that set any of these variables. Now that they are exported here they don't need to be set elsewhere.
`ANDROID_HOME` and `ANDROID_NDK_HOME`? sure